app-leechcraft/liblaretz/liblaretz-0.1.2.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Shared library to be used by the Laretz sync server and its clients"
+HOMEPAGE="https://leechcraft.org"
+SRC_URI="https://github.com/0xd34df00d/${PN#lib}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+inherit cmake-utils
+
+LICENSE="Boost-1.0"
+SLOT="0"
+KEYWORDS="amd64 x86"
+IUSE=""
+
+DEPEND="dev-libs/boost"
```

Comment:
Verify whether this gets linked, ie, is subslot binding necessary here? `:=`

Reply:
Awesome, thanks for catching this! Indeed it does!